src/routes/widgets/index.js

```diff
@@ -1,8 +1,15 @@
 import express from 'express';
 import { getWidget } from './handlers';
 
+const nr = require('newrelic');
+
 const router = express.Router();
 
-router.get('/:widgetId', getWidget);
+function nameWidgetTransactions(req, res, next) {
+  nr.setTransactionName(`GET /api/widgets/${req.params.widgetId}`);
```

Comment:
Not sure if it matters or is possible, but might be nice to have the name of the widget in addition to the ID.

Reply:
I had this concern too, but it turns out the widgetId is a good name, not an integer or something inscrutable. Example: `topicFrequencyGraph`.

Reply:
Ok perfect.